src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -503,61 +522,30 @@ const NetworkModificationNodeEditor = () => {
                 currentNode.id,
                 copiedModifications,
                 copyInfos
-            )
```

Comment:
because we dont need a response anymore, I think copyOrMoveModifications could use backendFetch rather than backendFetchText

Reply:
indeed, fixed